source/get-started/index.html.md.erb

```diff
@@ -0,0 +1,95 @@
+---
+title: Get started
+weight: 11
+---
+
+# Get started
+
+Get one GOV.UK Frontend component working in your application, so you can test everything works before you add more components or styles.
+
+You will need to do all the following steps to get your component working.
+
+1. Install GOV.UK Frontend.
+2. Add the HTML for a component to your application.
+3. Get the CSS working.
+4. Get the font and images working.
+5. Get the JavaScript working.
+
+## 1. Install
+
+[Install GOV.UK Frontend using npm](/installing_with_npm).
```

Comment:
It'd be great to rename the folders so that they're all kebab case, at the minute we have a mix of kebab case and snake case. Possibly makes sense to do this in a follow up PR? I don't mind doing it, if you prefer.

Reply:
Also, again we can avoid a 301 redirect by adding the trailing slash:

```suggestion
[Install GOV.UK Frontend using npm](/installing_with_npm/).
```

Reply:
Ah yes - sorry that's my mistake. You're right they should all be kebab case. I'll fix that up.

Will fix the 301s too!

Reply:
We're now consistent, but seem to be using snake_case not kebab-case.

I guess this is as much personal preference as anything, and I'm happy to defer to you on this one, but my personal preference would be for kebab-case, because:

- it's consistent with the way that we slug anchors (e.g. `/importing-css-assets-and-javascript/#font-and-image-assets` vs `/importing_css_assets_and_javascript/#font-and-image-assets`)
- it's consistent with the Design System (https://design-system.service.gov.uk/patterns/contact-a-department-or-service-team/)
- I think it's generally best practise? (Google recommends it anyway – https://support.google.com/webmasters/answer/76329!)

Reply:
Thanks Olli - not my day today. Everything should be kebab-case now. I even found one link where I'd used snake_case wrongly in a link to the Design System website!